### ISSUE TITLE
057: no longer adds all required keymaps to initrd

### ISSUE THREAD

**Issue by Polynomial-C** [dracutdevs/dracut#1859]
**Describe the bug**
After upgrading from `dracut-056` to `dracut-057` I can no longer successfully enter my LUKS password at boot.

**Distribution used**
Gentoo Linux

**Dracut version**
057

**Init system**
sysvinit + openrc

**To Reproduce**
```
# grep '^i18n_vars' /etc/dracut.conf
i18n_vars="/etc/conf.d/keymaps:keymap-KEYMAP /etc/conf.d/consolefont:consolefont-CONSOLEFONT-FONT /etc/rc.conf:UNICODE"
dracut -H /boot/initrd-$(uname -r) $(uname -r)
```

**Expected behavior**
From `056` initrd:
```
# lsinitrd /boot/initrd-5.15.51_01 | grep -F qwert
-rw-r--r--   1 root     root          456 Jun 17 11:57 usr/share/keymaps/i386/include/qwertz-layout.inc
drwxr-xr-x   1 root     root            0 Jun 30 08:15 usr/share/keymaps/i386/qwertz
-rw-r--r--   1 root     root          493 Jun 17 11:57 usr/share/keymaps/i386/qwertz/de-latin1-nodeadkeys.map
-rw-r--r--   1 root     root         2359 Jun 17 11:57 usr/share/keymaps/i386/qwertz/de-latin1.map
```
But in `057` initrd I only have:
```
# lsinitrd temp/initrd-5.15.51 | grep -F qwert
drwxr-xr-x   1 root     root            0 Jun 30 11:32 usr/share/keymaps/i386/qwertz
-rw-r--r--   1 root     root          493 Jun 17 11:57 usr/share/keymaps/i386/qwertz/de-latin1-nodeadkeys.map
```

**Comment by Polynomial-C** [comment]
Grepping for `keymaps` makes it even more obvious:
`056` initrd:
```
# lsinitrd /boot/initrd-5.15.51_01 | grep -F keymaps
drwxr-xr-x   1 root     root            0 Jun 30 08:15 usr/share/keymaps
drwxr-xr-x   1 root     root            0 Jun 30 08:15 usr/share/keymaps/i386
drwxr-xr-x   1 root     root            0 Jun 30 08:15 usr/share/keymaps/i386/include
-rw-r--r--   1 root     root          278 Jun 17 11:57 usr/share/keymaps/i386/include/compose.inc
-rw-r--r--   1 root     root          253 Jun 17 11:57 usr/share/keymaps/i386/include/euro2.map
-rw-r--r--   1 root     root         4353 Jun 17 11:57 usr/share/keymaps/i386/include/linux-keys-bare.inc
-rw-r--r--   1 root     root          746 Jun 17 11:57 usr/share/keymaps/i386/include/linux-with-alt-and-altgr.inc
-rw-r--r--   1 root     root          456 Jun 17 11:57 usr/share/keymaps/i386/include/qwertz-layout.inc
drwxr-xr-x   1 root     root            0 Jun 30 08:15 usr/share/keymaps/i386/qwertz
-rw-r--r--   1 root     root          493 Jun 17 11:57 usr/share/keymaps/i386/qwertz/de-latin1-nodeadkeys.map
-rw-r--r--   1 root     root         2359 Jun 17 11:57 usr/share/keymaps/i386/qwertz/de-latin1.map
drwxr-xr-x   1 root     root            0 Jun 30 08:15 usr/share/keymaps/include
-rw-r--r--   1 root     root          517 Jun 17 11:57 usr/share/keymaps/include/compose.8859_7
-rw-r--r--   1 root     root           23 Jun 17 11:57 usr/share/keymaps/include/compose.8859_8
-rw-r--r--   1 root     root         6030 Jun 17 11:57 usr/share/keymaps/include/compose.latin
-rw-r--r--   1 root     root         3889 Jun 17 11:57 usr/share/keymaps/include/compose.latin1
-rw-r--r--   1 root     root         3465 Jun 17 11:57 usr/share/keymaps/include/compose.latin2
-rw-r--r--   1 root     root         2037 Jun 17 11:57 usr/share/keymaps/include/compose.latin3
-rw-r--r--   1 root     root         2897 Jun 17 11:57 usr/share/keymaps/include/compose.latin4
```
`057` initrd:
```
# lsinitrd temp/initrd-5.15.51 | grep -F keymaps
drwxr-xr-x   1 root     root            0 Jun 30 11:32 usr/share/keymaps
drwxr-xr-x   1 root     root            0 Jun 30 11:32 usr/share/keymaps/i386
drwxr-xr-x   1 root     root            0 Jun 30 11:32 usr/share/keymaps/i386/qwertz
-rw-r--r--   1 root     root          493 Jun 17 11:57 usr/share/keymaps/i386/qwertz/de-latin1-nodeadkeys.map
```

**Comment by aafeijoo-suse** [comment]
Pending fix: #1856

**Comment by LaszloGombos** [comment]
Seems this can be closed now; fix is scheduled for dracut-058
